backend/logic/courses/courses.js

```diff
@@ -145,9 +147,26 @@ var addCourseOfferings = async courseOfferings => {
   }
 };
 
```

Comment:
Please specify an API description for the function
```suggestion
/**
 * Phases out a course that is no longer offered
 * @author Alex Lam
 * @param {string} courseCode
 * @returns true if successful
 * @throws error if MySQL connection failed
 *         invalid format course code if course code format is incorrect
 *         false if modification failed
*/
```

Reply:
I put it with the endpoint in the other file

Reply:
yeah but from i thought we were doing an API description for the endpoint and also one for the actual methods?

Reply:
whaat, its the same description though thats so redundant, I dont think we should do that

Reply:
The route might be calling more than one function, each function should have a description of what it does. Just like javadocs man
At least that's my opinion on it, it's minor thing.

Reply:
Ahh I see what you mean. Alright ill add it thanks